#include "names.h"

namespace visp_blobs_tracker
{
  std::string camera_info_topic("camera_info");
  std::string image_topic("image_raw");
  std::string status_topic("status");
  std::string object_position_topic("object_position");
  std::string object_des_position_topic("object_des_position");
  std::string object_cog_topic("object_cog");
  std::string code_message_topic("code_message");
  std::string init_service("init");


}
